app/message/write/_components/form.tsx

```diff
@@ -30,21 +30,35 @@ const MessageWriteForm = ({
       {
         id: 'content',
         validate: value => value.length > 0,
-        message: '내용을 입력해주세요',
+        message: ' ',
+      },
+      {
+        id: 'content',
+        validate: value => value.length <= 200,
```

Comment:
```suggestion
        validate: {length} => length <= 200,
```
구조 분해 할당으로 작성할 수도 있을 것 같아요.
위에 onSubmit과 validationRulesList에도 해볼 수 있을 것 같아요

Reply:
넵 수정해보겠습니다.!